File: FloridaShiduch/Models/Profile/Spouse.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using System.Data.Entity;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace FloridaShiduch.Models.Profile
{
    public class Spouse
    {
        [Key, ForeignKey("ApplicationUser")]
        public string Id { get; set; }
        public int MinAge { get; set; }
        public int MaxAge { get; set; }
        public int MinFeet { get; set; }
        public int MinInches { get; set; }
        public int MaxFeet { get; set; }
        public int MaxInches { get; set; }
        public bool AllowChildren { get; set; }
        public string TV { get; set; }
        public bool Relocate { get; set; }
        public bool Aliyah { get; set; }

        public virtual ApplicationUser ApplicationUser { get; set; }
    }
}

File: FloridaShiduch/Scripts/src/behaviors/spouse.js
define(['marionette'], function (Marionette) {
    'use strict';
    return Marionette.Behavior.extend({
        ui: {
            minAge: '#register-spouse-age-min',
            maxAge: '#register-spouse-age-max',
            minFeet: '#register-spouse-height-min-feet',
            minInches: '#register-spouse-height-min-inches',
            maxFeet: '#register-spouse-height-max-feet',
            maxInches: '#register-spouse-height-max-inches',
            maritalStatus: 'input[name="register-spouse-maritalstatus"]',
            allowChildren: 'input[name="register-spouse-children"]',
            hebrewEducationLevel: 'input[name="register-spouse-hebrew-education"]',
            secularEducationLevel: 'input[name="register-spouse-education"]',
            tv: 'input[name="register-spouse-tv"]',
            relocate: 'input[name="register-spouse-relocate"]',
            aliyah: 'input[name="register-spouse-aliyah"]'
        }
    });
});